src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/JvmUsageBucketThresholds.java

```diff
@@ -0,0 +1,26 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.summaries;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.summaries.HotResourceSummary.UsageBucket;
+
+public class JvmUsageBucketThresholds implements UsageBucketThresholds {
+  private StaticBucketThresholds youngGenPromotionRateThresholds;
+  private StaticBucketThresholds oldGenHeapUsageThresholds;
+
+  public JvmUsageBucketThresholds(StaticBucketThresholds youngGenPromotionRateThresholds,
+      StaticBucketThresholds oldGenHeapUsageThresholds) {
+    this.youngGenPromotionRateThresholds = youngGenPromotionRateThresholds;
+    this.oldGenHeapUsageThresholds = oldGenHeapUsageThresholds;
+  }
+
+  @Override
+  public UsageBucket computeBucket(HotResourceSummary resourceSummary) {
```

Comment:
same as above. I think this needs to be moved into flowunit

Reply:
Moved into the ClusterFlowUnit class